src/WebJobs.Script/Config/ConfigurationSectionNames.cs

```diff
@@ -25,5 +25,10 @@ public static class ConfigurationSectionNames
         public const string SequentialJobHostRestart = JobHost + ":sequentialRestart";
         public const string SendCanceledInvocationsToWorker = "sendCanceledInvocationsToWorker";
         public const string TelemetryMode = "telemetryMode";
+
+        /// <summary>
+        /// Determine if the host.json file was created by the system.
+        /// </summary>
+        public const string IsDefaultHostConfig = "isDefaultHostConfig";
```

Comment:
Doesn't this property need to be defined in ScriptApplicationHostOptions? That is where the summary comment should be instead of here. https://github.com/Azure/azure-functions-host/blob/5bac7614d792ff897daff3579623874b614ab8a0/src/WebJobs.Script/Config/ScriptJobHostOptions.cs#L122-L125